Dockerfile

```diff
@@ -10,7 +10,7 @@
 #
 
 # https://registry.access.redhat.com/ubi8/go-toolset
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19.13-2 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.20.10-3 as builder
```

Comment:
hopefully moving to 1.20 won't break anything and only fix CVEs.

see also thread in https://github.com/eclipse-che/che-machine-exec/pull/273#pullrequestreview-1735752995